packages/components/src/components/banner/banner-styles.ts

```diff
@@ -33,12 +33,16 @@ export const getComponentCss = (isOpen: boolean): string => {
         display: 'block',
         ...addImportantToEachRule({
           ...getBannerPopoverResetStyles(),
+          top: 'auto',
```

Comment:
Maybe better to overwrite the UA inset style in `getBannerPopoverResetStyles()` e.g. `inset: auto`